test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1

```diff
@@ -116,13 +120,31 @@ Describe "Start-Process" -Tags @("Feature") {
 
     It "Should be able to use the -WhatIf switch without performing the actual action" {
         $pingOutput = Join-Path $TestDrive "pingOutput.txt"
-        { Start-Process -Wait $pingCommand -ArgumentList $pingParam -RedirectStandardOutput $pingOutput -WhatIf -ErrorAction Stop } | Should -Not -Throw
+        { Start-Process -Wait $pingCommand -ArgumentList $pingParam -RedirectStandardOutput $pingOutput -WhatIf -ErrorAction Stop  @extraArgs} | Should -Not -Throw
         $pingOutput | Should -Not -Exist
     }
 
     It "Should return null when using -WhatIf switch with -PassThru" {
         Start-Process $pingCommand -ArgumentList $pingParam -PassThru -WhatIf | Should -BeNullOrEmpty
-   }
+    }
+
+    It 'Should run without errors when -ArgumentList is $null' {
+         $process = Start-Process $pingCommand -ArgumentList $null -PassThru @extraArgs
+         $process.Length      | Should -Be 1
+         $process.Id          | Should -BeGreaterThan 1
```

Comment:
Is this check required?

Reply:
These are basic sanity tests to make sure the process was created and is running (has an ID).

Reply:
If the process is not started the `$process` is `$null`.